content/en/docs/concepts/security/index.md

```diff
@@ -108,12 +108,12 @@ platforms:
    service account refers to the existing service account just like the
    identities that the customer's Identity Directory manages.
 
-## Public Key Infrastructure (PKI) {#pki}
+## Identity Issuance {#pki}
 
-The Istio PKI securely provisions strong identities
-to every workload with X.509 certificates. To automate key and certificate
-rotation at scale, the PKI runs an Istio agent alongside each Envoy proxy for
-certificate and key provisioning. The following diagram shows the identity
+The Istio securely provisions strong identities
+to every workload with X.509 certificates. The Istio agent running alongside 
```

Comment:
```suggestion
to every workload with X.509 certificates. The istiod and istio-agents running alongside 
```

Reply:
It looks like elsewhere, we use backticks around "istiod".  Do you think it should be done here, too?

Reply:
```suggestion
to every workload with X.509 certificates. Istio agents, running alongside each Envoy proxy, 
```